ZenPacks/zenoss/LinuxMonitor/FileSystem.py

```diff
@@ -168,69 +150,31 @@ def getStorageServer(self):
             except AttributeError:
                 pass
 
+    def getStorageDevice(self):
+        """Get the storage device that contains this fs.
 
-class IFileSystemInfo(IBaseFileSystemInfo):
-
-    """IInfo interface for FileSystem.
-
-    This is used for JSON API definition. Fields described here are what will
-    appear on instance's component details panel.
-
-    """
-
-    logicalVolume = schema.Entity(
-        title=_t(u"LVM Logical Volume"),
-        group="Details",
-        readonly=True)
-
-    blockDevice = schema.Entity(
-        title=_t(u"Block Device"),
-        group="Details",
-        readonly=True)
-
-
-class FileSystemInfo(BaseFileSystemInfo):
-
-    """Info adapter for FileSystem.
-
-    This is used for API implementation and JSON serialization. Properties
-    implemented here will be available through the JSON API.
-
-    """
-
-    implements(IFileSystemInfo)
-
-    server_name = ProxyProperty('server_name')
-
-    @property
-    @info
-    def logicalVolume(self):
-        return self._object.logicalVolume()
+           :return: An html entity
+           :rtype:  string
+        """
 
-    @property
-    @info
-    def blockDevice(self):
-        return self._object.blockDevice()
+        storage = self.storageDevice
+        storage = super(FileSystem, self).storageDevice
```

Comment:
This line shadows the previous line. There's no reason to have one of them. Given that I don't think there's a local override of self.storageDevice, using just self.storageDevice seems like the right thing to do.